learn/educators/python.md

```diff
@@ -0,0 +1,98 @@
+---
+Order: 4
+Area: educators
+TOCTitle: Python
+ContentId: e58523d5-e243-45c2-8306-7114077d81ae
+PageTitle: Python in Visual Studio Code
+DateApproved: 02/17/2021
+MetaDescription: Python in Visual Studio Code
+---
+
+# Python
+
+Visual Studio Code is free professional editor that fully supports Python and other features like real-time collaboration. It's highly customizable to support your classroom the way you like to teach.
+
+> "[Visual Studio Code] is the best balance of authenticity and accessibility. Plus, [Visual Studio Code] works on every OS!" - Professor Zachary Dodds from Harvey Mudd College
+
+Read below for recommendations for extensions, settings, and links to free lessons that you can use in your classes.
+
+## Python Extension Pack for the classroom
```

Comment:
"Python Extension Pack"

Reply:
Should you go right to the case study? That is the value prop driver.
I'm not sure why Extension Pack is here and Coding Pack is below and it may confuse readers. 
You may want to put them together in a shared section (H2 under something like "Setup options") and explain why an educator would use/recommend one or the other.